admin/admin_site/views.py

```diff
@@ -1,15 +1,30 @@
 from django.shortcuts import render, redirect
 from .models import *
 from .forms import *
+from django.db.models import Q
```

Comment:
파이썬에는 pep8 이라는 스타일 가이드란게 있습니다.

그 중에서 import에 관련한 가이드가 존재하는데요,  
import 대상에 따라 순서를 정해놓고 사용하고 있습니다.


```suggestion
from django.db.models import Q
from django.shortcuts import render, redirect

from .forms import *
from .models import *

```
와 같이 외부 패키지를 먼저 명시한 뒤 내부 모듈을 가져오시면 좋을 것 같아요.

또한 import *와 같이 모듈을 가져오는 부분은 안티패턴에 해당합니다.
귀찮더라도 특정 클래스 또는 함수를 명시해서 가져오도록 수정하면 좋을 것 같아요.